src/components/modal/mute/modal.scss

```diff
@@ -31,7 +32,7 @@
         padding-top: 32px;
     }
     .bottom-img {
-        width: 100%;
+        width: 380px;
```

Comment:
Is there a way not to have an absolute pixel value here - it will may cause issues with responsive/small screens.

Reply:
I was hoping I could keep it at 100%, but, the way the html is currently structured, it ends up being too small. 
I haven't done a responsiveness pass on this modal at all yet yet. What do you think doing that in a follow up pass once it is easier to test on staging?

Reply:
Yes, let's do this as a follow up!